# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/cookiestore/PCookieStore.ipdl
# Commit: 56cce39fb13c
# Full Hash: 56cce39fb13cd0623644bf9d40cbcec3473a93d1
# Author: edgul <edgul@mozilla.com>
# Date: 2025-06-03 21:32:22
# Description:
#   Bug 1968495 - Pass NotNull nsURIs across cookiestore IPC. r=baku,cookie-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251177
# ==============================================================================

diff -r e93a3c226679 -r 56cce39fb13c dom/cookiestore/PCookieStore.ipdl
--- a/dom/cookiestore/PCookieStore.ipdl	Tue Jun 03 13:21:06 2025 +0000
+++ b/dom/cookiestore/PCookieStore.ipdl	Tue Jun 03 13:21:35 2025 +0000
@@ -33,7 +33,7 @@
   manager PBackground;
 
 parent:
-  async GetRequest(nullable nsIURI cookieURI,
+  async GetRequest(nsIURI cookieURI,
                    OriginAttributes attrs,
                    OriginAttributes? partitionedAttrs,
                    bool thirdPartyContext,
@@ -45,7 +45,7 @@
                    nsCString path,
                    bool onlyFirstMatch) returns (CookieStruct[] data);
 
-  async SetRequest(nullable nsIURI cookieURI,
+  async SetRequest(nsIURI cookieURI,
                    OriginAttributes attrs,
                    bool thirdPartyContext,
                    bool partitionForeign,
@@ -61,7 +61,7 @@
                    bool partitioned,
                    nsID operationId) returns (bool waitForNotification);
 
-  async DeleteRequest(nullable nsIURI cookieURI,
+  async DeleteRequest(nsIURI cookieURI,
                       OriginAttributes attrs,
                       bool thirdPartyContext,
                       bool partitionForeign,
